File: src/main/scala/org/biacode/biacentric/biap2p/configuration/Biap2pAnnotationDrivenConfiguration.scala
package org.biacode.biacentric.biap2p.configuration

import org.springframework.context.annotation.{ComponentScan, Configuration, PropertySource}

/**
  * Created by Arthur Asatryan.
  * Date: 5/6/18
  * Time: 5:17 PM
  */
@Configuration
@ComponentScan(Array("org.biacode.biacentric.biap2p"))
@PropertySource(value = Array(
  "classpath:netty-default.properties",
  "classpath:netty-custom.properties"
), ignoreResourceNotFound = true)
class Biap2pAnnotationDrivenConfiguration

File: src/main/java/org/biacode/biacentric/biap2p/network/configuration/Biap2pAnnotationDrivenConfiguration.java
package org.biacode.biacentric.biap2p.network.configuration;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.PropertySource;

/**
 * Created by Arthur Asatryan.
 * Date: 5/12/18
 * Time: 5:29 PM
 */
@Configuration
@ComponentScan("org.biacode.biacentric.biap2p")
@PropertySource(value = {
        "classpath:netty-default.properties",
        "classpath:netty-custom.properties"
}, ignoreResourceNotFound = true)
public class Biap2pAnnotationDrivenConfiguration {
}
